porchlightapi/settings.py

```diff
@@ -35,5 +36,14 @@
                                'PORCHLIGHT_JSON_FILE',
                                PORCHLIGHT_JSON_FILE_DEFAULT)
 
+PORCHLIGHT_GITHUB_AUTH_DEFAULT = None
+PORCHLIGHT_GITHUB_AUTH = getattr(settings,
+                                 'PORCHLIGHT_GITHUB_AUTH',
+                                 PORCHLIGHT_GITHUB_AUTH_DEFAULT)
+
+PORCHLIGHT_GITHUB_TAG_PATTERN_DEFAULT = r'^v[.0-9]+'
```

Comment:
@marcesher That's certainly the question we've been struggling with: how to know when a commit is deployed. Part of what I'm trying to do here is pull in and formalize some of the quick-and-dirty code I wrote to pull in some data to demo the app, so we *can* use git tags if we want. But @Scotchester is right, for that demo data I used all tags, with no particular pattern (because v[.0-9]+ didn't match all version tags). That said, having a specific pattern seems to be a saner public default.

I agree this is a question of deployment strategy for the dev ops team internally. I'm just a monkey slinging code at the wall right now :)